packages/makeup-modal/index.js

```diff
@@ -23,6 +26,7 @@ function unmodal() {
 function modal(el, options) {
   unmodal();
   modalEl = el;
+  makeupHoist.hoist(modalEl);
```

Comment:
We should make this configurable (i.e passed in via options). Default should be false for backwards compatibility.